blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,153 @@
+execution of a command. When the command is done, you can run your build with `bloop`.
+
+This configuration file is a properties file that we plan on migrating from. In the next weeks, we'd
+like to come up with the design of a configuration file that we can reuse across different tools in
+the Scala ecosystem. Given how common this kind of information is required, we believe that settling
+on an agreed upon configuration format is the way to go.
+
+In the future, we want to explore making `bloop` a server you can ask compilation and test-related
+information from different language servers, presentation compilers and linters.
+
+## Numbers on compilation performance
+
+`bloop` has been created out of the observation that Scala compile times slow down within `sbt` in
+comparison with isolated benchmarks. This observation was at the beginning just an intuition, but
+when we sat down to measure a prototype we did see a significant difference.
+
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr"> Scala Tooling has got to
+a point where people say "compiler performance" and what they really mean is "sbt performance".
+</p>&mdash; Jorge (@jvican) <a href="https://twitter.com/jvican/status/928601470232129536">9th
+November 2017</a></blockquote><script async src="//platform.twitter.com/widgets.js"
+charset="utf-8"></script>
+
+We'd like to share some of our numbers with you.
+
+| Project name      | sbt (version) | bloop | speedup |
+| ----------------- | ------------- | ----- | ------- |
+| sbt/sbt           | 21s (1.0.4)   | 15s   | 1.17    |
+| guardian/frontend | 37s (0.13.16) | 28s   | 1.32    |
+| apache/spark      | 159s (0.13.16)| 107s  | 1.4     |
+| scala/scala       | 65s (0.13.16) | 50s   | 1.14    |
+
+These numbers have been obtained in an isolated machine, measuring at the fifth hot clean-compile
+iteration (of all the projects and subprojects of the build without counting the tests) and with 2
+gigabytes of heap for both sbt and bloop. When benchmarking sbt, we have made sure that dependency
+resolution and formatting are not measured.
+
+There are two important observations to make from these numbers:
+
+- In practice, sbt is slower than the previous numbers because dependency resolution is often
+  triggered by compilation, slowing down the start of the compile.
+- Some of the benchmarked builds use sbt `0.13.16` instead of `1.0.4`. `bloop` would be even faster
+  if these builds were migrated to `1.0.4`. The refined precision that Zinc 1.0 offers in its
+  incremental mode has a price of ~3% compared to Zinc 0.13.16.
+
+## Conclusion
+
+We're happy to announce `bloop`, a tool with which sbt users can compile their projects faster and
+execute tests without being tied to an sbt instance. Users of other build tools will get support for
+`bloop` in the upcoming weeks.
+
+`Bloop` is quickly improving to deliver a snappy developer experience to those developers writing
```

Comment:
I'm not sure that this was proven though?

Reply:
Not proven, bad wording, thanks for this catch.